src/components/Article/Article.jsx

```diff
@@ -1,5 +1,68 @@
 // import a css file containig article styles
-
+import './Article.css'
 // Create an Article function returning the HTML of article block
-
+const Article = () => {
+  return (
+    <article className="article">
```

Comment:
You don't need to use return here
```suggestion
const Article = () => (
    <article className="article">
```